package.json

```diff
@@ -18,6 +18,8 @@
   "homepage": "https://github.com/Linkurious/squanchy-server#readme",
   "dependencies": {
     "basic-auth": "2.0.1",
+    "compressible": "^2.0.18",
```

Comment:
Please avoid non-strict-version dependencies in package.json, they often lead to "npm install works for you but not for me" issues.